assets/main.css

```diff
@@ -0,0 +1,703 @@
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-position: center;
+  color: #ec5242;
+  font-weight: 900;
+  font-size: 1.95rem;
+  background-size: cover;
+}
+
+.title-and-description .description-text {
+  border: 2px solid white;
+  padding: 1rem;
+  background-color: rgb(211, 211, 211, 0.2);
+}
+
+.title-and-description .program-time {
+  font-weight: 900;
+}
+
+.title-and-description .program-place {
+  font-weight: 400;
+}
+
+/* *********** mainprogram section ****** */
+
+.main-program {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 2rem 1rem;
+  background-color: var(--main);
+  color: var(--third);
+  background-image: url("./images/pattern-1.png");
+  background-position: center;
+  background-repeat: repeat;
+  background-size: 10%;
+  backdrop-filter: blur(8px);
+}
+
+.main-program .main-program-title {
+  padding: 1rem;
+  font-size: 1.5rem;
+}
+
+.main-program-title::after {
+  content: "";
+  position: absolute;
+  top: 5.5rem;
+  left: 50%;
+  width: 3rem;
+  height: 1px;
+  transform: translate(-50%, -50%);
+  background-color: #ec5242;
+}
+
+.main-program .program-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1rem;
+  align-items: center;
+}
+
+.main-program .program-container .content {
+  padding: 1rem 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  background-color: rgb(73, 79, 92, 0.9);
+  width: 100%;
+}
+
+.content .icon {
+  width: 30%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-and-description-container img {
+  width: 30%;
+  z-index: 2;
+  padding: 0.5rem;
+  object-fit: cover;
+}
+
+.content .icon img {
+  width: 100%;
+}
+
+.content .title {
+  color: var(--second);
+  font-size: 1.5rem;
+}
+
+.main-program .join-btn {
+  background-color: var(--second);
+  color: var(--third);
+  outline: none;
+  border: none;
+  padding: 2rem;
+  font-size: 1.2rem;
+}
+
+/* *********** featured speakers section */
+
+.featured-speaker {
+  padding: 1rem 1rem;
+  background-color: #fff;
+}
+
+.title-container {
+  width: 100%;
+  padding: 1rem;
+  text-align: center;
+  color: var(--main);
+}
+
+.title-container .title-text {
+  position: relative;
+  font-size: 1.5rem;
+}
+
+.title-container .title-text::after {
+  content: "";
+  position: absolute;
+  top: 2.5rem;
+  left: 50%;
+  width: 3rem;
+  height: 1px;
+  transform: translate(-50%, -50%);
+  background-color: #ec5242;
+}
+
+.dynamic-content-container {
+  padding: 2rem 0;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+
+.image-and-description-container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.image-and-description-container .transparent-image {
```

Comment:
- [ ] `required`
- [ ] Kindly fix the broken the background for the image 🥂 
- [ ] see screenshot below

![image](https://user-images.githubusercontent.com/66230306/226145271-eca99803-5ecf-4363-942c-80cece55ebe3.png)